puppet/modules/slave/manifests/init.pp

```diff
@@ -281,12 +264,9 @@
   # Needed by foreman_openscap gem dependency OpenSCAP
   if $::osfamily == 'RedHat' {
     yumrepo { 'isimluk-openscap':
-      enabled     => 1,
-      gpgcheck    => 0,
-      descr       => 'isimluk-openscap',
-      baseurl     => "http://copr-be.cloud.fedoraproject.org/results/isimluk/OpenSCAP/epel-${::operatingsystemmajrelease}-\$basearch/",
-      includepkgs => ['openscap'],
-    } ->
+      rensure     => absent,
```

Comment:
Typo I assume rensure/ensure

Reply:
I did see it in a later commit but looks like I didn't push that